.github/workflows/test-coverage.yaml

```diff
@@ -35,7 +38,7 @@ jobs:
           covr::to_cobertura(cov)
         shell: Rscript {0}
 
-      - uses: codecov/codecov-action@v4.5
+      - uses: codecov/codecov-action@v4
```

Comment:
downgrade?

Reply:
For some reason it does not work.
\+ looking for v4 will already get latest v4.x
\+ It wasn't the issue